aten/src/ATen/native/native_functions.yaml

```diff
@@ -6501,6 +6501,20 @@
   dispatch:
     CPU, CUDA: erfinv_out
 
+- func: erfcx(Tensor self) -> Tensor
```

Comment:
Take a look at the i0e entries and let's add this function to the torch.special namespace:

https://github.com/pytorch/pytorch/blob/d09abf004cc16f8fd5f320e3d5d07c383c174ea7/aten/src/ATen/native/native_functions.yaml#L9316

Reply:
Added special_erfcx below.